x-pack/plugins/security_solution/public/flyout/left/components/session_view.tsx

```diff
@@ -7,16 +7,45 @@
 
 import type { FC } from 'react';
 import React from 'react';
-import { EuiText } from '@elastic/eui';
-import { SESSION_VIEW_TEST_ID } from './test_ids';
+import { EuiEmptyPrompt } from '@elastic/eui';
+import { ERROR_MESSAGE, ERROR_TITLE } from '../../shared/translations';
+import { SESSION_VIEW_ERROR_MESSAGE } from './translations';
+import { SESSION_VIEW_ERROR_TEST_ID, SESSION_VIEW_TEST_ID } from './test_ids';
+import { useKibana } from '../../../common/lib/kibana';
+import { useLeftPanelContext } from '../context';
 
 export const SESSION_VIEW_ID = 'session_view';
+const SESSION_ENTITY_ID = 'process.entry_leader.entity_id';
 
 /**
  * Session view displayed in the document details expandable flyout left section under the Visualize tab
  */
 export const SessionView: FC = () => {
-  return <EuiText data-test-subj={SESSION_VIEW_TEST_ID}>{'Session view'}</EuiText>;
+  const { sessionView } = useKibana().services;
+  const { getFieldsData } = useLeftPanelContext();
+
+  const sessionEntityId = getFieldsData(SESSION_ENTITY_ID) as string;
```

Comment:
instead of casting here, maybe reuse `getField` method [here](https://github.com/elastic/kibana/pull/154196/files#diff-998dc1f2f6c2f703bb6ab00309053962f5f65f602d6069ef8ad8822dac5a14bfR14)?

Reply:
I don't know how I missed that, thank you!